test_requirements.txt

```diff
@@ -1 +1,5 @@
-git+https://github.com/aymgal/coolest@main#egg=coolest
\ No newline at end of file
+git+https://github.com/aymgal/coolest@main#egg=coolest
+hmf
+pytest
+scipy
```

Comment:
do you need scipy and tqdm be mentioned in the test_requirements since they are also in the main requirements?